Common/Inc/CommonDataTypes.hpp

```diff
@@ -18,6 +21,137 @@ typedef struct {
     float longitudeSpeed; //m/s
     double track; // degrees
     double heading; //degrees
-} LosSFData;
+};
+
+} // Namespace LOS
+
+namespace PM {
+
+struct JetsonToZpMovementCommand {
+  float x;
+  float y;
+  float z;
+  float heading;
+};
+
+struct LandingInitiationCommand {
+    bool start_landing; 
+};
+
+struct JetsonMovementRequest {
+    bool request;
+};
+
+struct TelemWaypoint {
+    uint8_t waypoint_id;
+    double longitude;
+    double latitude;
+    double altitude;
+};
+
+struct WaypointsCommand {
+    uint8_t num_waypoints; // number of valid waypoints in the list 
+    TelemWaypoint waypoints[5]; 
+};
+
+struct PM_AM_Commands {
+  // heading unit vector and magnitude
+  float dist_x; 
+  float dist_y; 
+  float dist_z; 
+  float magnitude; // Magnitude distance to waypoint target
+  float heading; // heading at target waypoint
+  double velocity_target; // Target velocity of drone approaching target
+}; 
+
+struct SM_PM_Commands {
+    WaypointsCommand telemetry_commands;
+    JetsonToZpMovementCommand jetson_commands; 
+    LandingInitiationCommand landing_initiation; 
+    LOS::LosSFData sf_data;
+};
+
+struct PM_SM_Commands {
+    JetsonMovementRequest jetson_movement_req;
+};
+
+} // Namespace PM
+
+namespace AM {
+
+struct AM_SM_Commands {
+    bool ok;
+};
+
+struct SM_AM_Commands {
+    uint8_t motor_outputs[12];
+};
+
+struct PIDValues {
+    double P;
+    double I;
+    double D;
+    double A;
+};
+
+struct PIDControllerValues {
+    PIDValues axes[6];
+};
+
+struct GroundStationPIDSetCommand {
+    uint8_t controller;
+    uint8_t axis;
+    PIDValues values;
+};
+
+struct GroundStationPIDSetResponse {
+    uint8_t controller_number;
+    PIDControllerValues controller;
+};
+
+struct ActuatorConfig {
+    uint8_t channel = UINT8_MAX;
+    StateMix stateMix;
```

Comment:
state_mix?

or stateMix pls double check our style guide i forget.

Reply:
should be state_mix:

in general: snake_case for variables, camelCase for functions

Reply:
whoops, I yoinked these from the AM code and assumed it was already following style guide, I'll fix